packages/extension/src/providers/ethereum/ui/send-transaction/index.vue

```diff
@@ -356,7 +357,7 @@ const setTransactionFees = (tx: Transaction) => {
 };
 
 const setBaseCosts = () => {
-  const web3 = new Web3Eth(props.network.node);
+  const web3 = new Web3Eth(new HoprProvider(props.network.node) as any);
```

Comment:
HoprProvider??

Reply:
Project does not have proper linting or even build - it was not detected by processes, but fixed. :)